pack/tfa/hote_encounter.json

```diff
@@ -27,168 +28,237 @@
         "code": "04208",
         "doom": 5,
         "double_sided": true,
         "encounter_code": "pillars_of_judgment",
+        "encounter_position": 2,
         "faction_code": "mythos",
         "flavor": "The day advances quickly. Through the thick canopy of trees, you can tell that the sun is starting to sink closer to the horizon. To the east, the sky grows dim. Soon the jungle will be covered in darkness, and you will have to stop and make camp for the night.",
         "illustrator": "Dimitri Bielak",
         "name": "Setting Sun",
         "pack_code": "hote",
         "position": 208,
         "quantity": 1,
-        "text": "[action]: <b>Explore. </b>Draw the top card of the exploration deck. If it is a connecting location, put it into play and move to it.",
+        "stage": 2,
+        "text": "[action]: <b>Explore.</b> Draw the top card of the exploration deck. If it is a connecting location, put it into play and move to it.",
         "type_code": "agenda"
     },
     {
-        "back_text": "<b>Spawn</b> - Mouth of K'n-yan.<br/>Alert. Hunter. Massive.<br/>Cannot be defeated. Cannot make attacks of opportunity.<br/><b>Forced</b> - After a pillar token is placed on the Mouth of K'n-yan: Exhaust The Winged Serpent. It does not ready during the upkeep phase this round.",
+        "back_link": "04209b",
         "clues": 2,
-        "code": "04209",
-        "double_sided": true,
+        "code": "04209a",
         "encounter_code": "pillars_of_judgment",
+        "encounter_position": 3,
         "faction_code": "mythos",
         "flavor": "Six stone pillars surround the entrance to the cavern, each carved with a series of uncanny grooves and hieroglyphs that form an intricate pattern. Touching them causes the grooves to glow brightly, and twisting them causes the patterns to shift and change. Perhaps these patterns match the glyphs found in other regions of the jungle?",
-        "illustrator": "Stephen Somers",
+        "illustrator": "Michele Frigo",
         "name": "Search for the Pattern",
         "pack_code": "hote",
         "position": 209,
         "quantity": 1,
+        "stage": 1,
         "type_code": "act"
     },
     {
-        "back_text": "<em class='bbc'>With a deep rumbling that shakes the ground outside, something inside the cavern shifts. In a distinct sequence, the six obelisks slowly descend into the ground, twisting and turning. The serpent creature that guarded the cavern lets out a shriek of anger and hate before flying off and vanishing behind the treeline. One by one, the stone pillars disappear into the dirt, and the Earth stops shaking.<br/></em><b>(→R1)</b>",
+        "code": "04209b",
+        "encounter_code": "pillars_of_judgment",
+        "encounter_position": 3,
+        "enemy_damage": 1,
+        "enemy_evade": 5,
+        "enemy_fight": 8,
+        "enemy_horror": 1,
+        "faction_code": "mythos",
+        "health": 0,
```

Comment:
I think 'null' is ok for blank healths?

Reply:
Changed it over to `null`, unless you mean I should leave it out all together?